### ISSUE TITLE
New smile [suggestion]

### ISSUE THREAD

**Issue by Nikv2005** [Monika-After-Story/MonikaModDev#9012]
Just a new smile for monika like in this artwork![ZpK_kRzhbEQ.jpg](https://user-images.githubusercontent.com/85229914/160352038-b757b92a-5db1-4c00-bf44-61524b6349c5.jpg). This smile does not replace any current smiles in game , its just a new one.

**Comment by kkrosie123** [comment]
I made something kinda similar to this, just not very wide is all.

**Comment by kkrosie123** [comment]
![image](https://user-images.githubusercontent.com/52429850/172057798-edf2f534-1f78-47c6-b45d-378c7cd6c1b7.png)

There are actually quite a few expressions I've made, just no implementation yet. Soon :tm: kinda thing, haha.

**Comment by Nikv2005** [comment]
> ![image](https://user-images.githubusercontent.com/52429850/172057798-edf2f534-1f78-47c6-b45d-378c7cd6c1b7.png)
> 
> There are actually quite a few expressions I've made, just no implementation yet. Soon :tm: kinda thing, haha.
> 
> But is that what you're looking for or do you want something more similar to the picture?

Oh my god, it's perfect! This should be in game!

**Comment by kkrosie123** [comment]
Ahaha, thanks! Looks like this is just awaiting implementation then.

**Comment by Nikv2005** [comment]
> Ahaha, thanks! Looks like this is just awaiting implementation then.

Yes , the main problem that it can take a thousand years to implement(... But I think this thing be useful for submod. Anyway... There is a way to get this!? Im lazy to draw myself this thing, also i need to a lot of time to learn drawing.

**Comment by AsuraShun** [comment]
Genius idea, really, she'd look even more beautiful, if that's even possible, with a wider variety of smiles
